Dockerfile

```diff
@@ -1,5 +1,6 @@
-FROM rocker/verse:4.0.4
-MAINTAINER support@civisanalytics.com
+ARG BUILDPLATFORM=linux/x86_64
+FROM --platform=${BUILDPLATFORM} rocker/verse:4.4.1
```

Comment:
Since Platform requires linux/x86_64 and since specifying the platform is needed for building an image on an arm64 apple machine, it seemed good to specify it here.